### PR TITLE
FIX: Load firmware online button not enabled if not in expert mode

### PR DESCRIPTION
**Pull request by blckmn**
As per the label on the tin.

If the expert mode was not enabled, and the cloud build was true, then the load firmware online would not be enabled.

A work around is to select a non-cloud build firmware option, and then reselect the cloud build firmware. This is not ideal for users - so this is considered an urgent fix.

### CODE REVIEW

**Review by sugaarK — APPROVED** [review]

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Review by Asizon — APPROVED** [review]

**Review by haslinghuis — APPROVED** [review]

**Pull request merged**
